src/HereAuth/HereAuth.php

```diff
@@ -221,7 +221,7 @@ public function onDisable(){
 	public function startUser(Player $player){
 		$this->database->loadFor($player->getName(), $player->getId());
 		if($player->spawned){
-			$player->sendMessage("[HereAuth] Your account data are being loaded. Please wait patiently; it shouldn't take long.");
+			$player->sendMessage("[HereAuth] Your account data is being loaded. Please wait patiently; it shouldn't take long.");
```

Comment:
@SOF3 Yes, you're strictly following the rules of grammar. But to majority of English speakers nowadays, it wouldn't make sense. I believe that whatever is used more should be used, and which, in this case, is "data is". However, this is up to @PEMapModder , to decide. https://www.theguardian.com/news/datablog/2010/jul/16/data-plural-singular